src/main/java/seedu/address/logic/commands/ViewCommand.java

```diff
@@ -0,0 +1,49 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.function.Predicate;
+
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+
+
+
+
```

Comment:
Small coding standard issue but too many spaces left. Please check other areas to ensure even spacings for better coding standard